Readme.md

```diff
@@ -0,0 +1,512 @@
+Conversation opened. 2 messages. All messages read.
+
+Skip to content
+Using Gmail with screen readers
+in:sent
+2 of 66
+(no subject)
+
+Akinmola Omotolani
+Wed, 26 Apr, 08:23 (12 days ago)
+<a name="readme-top"></a> <div align="center"> <h3><b>Hello Microverse</b></h3> </div> <!-- TABLE OF CONTENTS --> # 📗 Table of Contents - [📖 About the Project
+
+Akinmola Omotolani <akinmolaomotolani@gmail.com>
+Wed, 26 Apr, 08:40 (12 days ago)
+to abdoulramanediallo44
+
+---------- Forwarded message ---------
+From: Akinmola Omotolani <akinmolaomotolani@gmail.com>
+Date: Wed, 26 Apr 2023 at 08:23
+Subject:
+To: <kal2403199255@gmail.com>
+
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-microverse-project] <a name="setting Up a linters"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello-microverse-project]** is a project to set up html-css linters in order to be able to detect bugs in our code
```

Comment:
I think it would be better if you remove the comment.
It is the line that starts with '>'.
This will be helpful in making your Readme clean and professional